src/Arcus.Scripting.ApiManagement/Scripts/Import-AzApiManagementApiPolicy.ps1

```diff
@@ -12,7 +12,7 @@ if ($apim -eq $null) {
 $apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroupName -ServiceName $ServiceName
 
 Write-Host "Updating policy of API '$ApiId'"
-$result = Set-AzApiManagementPolicy -Context $apimContext -ApiId $ApiId -PolicyFilePath $policyFilePath
+$result = Set-AzApiManagementPolicy -Context $apimContext -ApiId $ApiId -PolicyFilePath $PolicyFilePath -PassThru
```

Comment:
Super found! 👍

Reply:
Today I Learned.

Didn't know what this PassThru parameter was about, and actually I find it quite strange that you need to specify that to get a return value. :?
What is more strange:  According to what I read, the PassThru parameter specifies that the newly created object (in this case, the policy) must be returned.   However, the documentation mentions that a boolean is outputted :?

> Well, ML, as you can see, using the passthru parameter forces Windows PowerShell to go ahead and pass newly created or modified objects instead of hiding them. By knowing when to use and not to use the parameter, great flexibility is gained.

Reply:
Didn't know this as well honestely, the microsoft docs specifically stated a boolean should be returned, but it is really only returned when using the `-PassThru` param... Took a while to find that 😄

Reply:
According to the documentation / explanation I copied above, I'd suppose that an object that represents the APIM policy should be returned :P